contracts/access/manager/AccessManager.sol

```diff
@@ -0,0 +1,558 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {IAccessManager} from "./IAccessManager.sol";
+import {IManaged} from "./IManaged.sol";
+import {Address} from "../../utils/Address.sol";
+import {Context} from "../../utils/Context.sol";
+import {Time} from "../../utils/types/Time.sol";
+
+contract AccessManager is Context, IAccessManager {
```

Comment:
It's been difficult to follow the naming because I think we have a few different concepts here, such as `delay`, `operation`, `action`, `access`, `group`, etc. I'd like to suggest the following:

- Use `groupId` instead of `group` for `uint256` types identifying a group (Group is already a struct)
- Use `operation` uniformly instead of `action`

Reply:
Done, 
I also renamed `id` to `operationId` where relevant